tests/tools/nnapi_test/src/nnapi_test.cc

```diff
@@ -57,7 +57,9 @@ int main(const int argc, char **argv)
     {
       seed = std::atoi(argv[2]);
     }
-    return nnfw::tflite::RandomTestRunner::make(seed).run(builder);
+    auto runner = nnfw::tflite::RandomTestRunner::make(seed);
+    runner.compile(builder);
+    return runner.run(2);
```

Comment:
I have two concern

- It seems strange `nnapi_test` itself always run twice
- It increases CI test time

Reply:
> It seems strange `nnapi_test` itself always run twice

Right, I will change running count to parameter of `nnapi_test`

> It increases CI test time

I think it is not big because it consumes only execution time once more.